AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -570,6 +570,7 @@ hgtv.com#@#.showads
 !
 !
 zhangxinxu.com#?#body > *:not(div):not(script):matches-css(width:336px)
+zhangxinxu.com#?#[class^="col-left"] a[rel*="nofollow"], [class^="col-right"] a[rel*="nofollow"]
```

Comment:
We don't add rules with several selectors together, also there is no need for using extended-css `#?#`, wrong section for the rule. It's self-promo.
See how it should be https://github.com/AdguardTeam/AdguardFilters/commit/706a8979619d8ee55e5d317b35f830fb0fd228a7

Reply:
Okay, but apparently it uses "dynamic class" for the entire page with only ads and forces `display:block !important`, isn't that an anti-adblock? Or is there another reason?

Reply:
> forces `display:block !important`

In this case you can use `#?#` or other variants to avoid this. I exchanged one rule for apps.

> Okay, but apparently it uses "dynamic class" for the entire page with only ads and forces `display:block !important`, isn't that an anti-adblock? Or is there another reason?

You can see those elements without AdGuard, so no. I suppose he uses this property to avoid blocking - mostly, the basic rules `##` don't work against this.

Reply:
I get it, thanks